modules/observe/proc-admin-enabling-metrics-ocp-operator.adoc

```diff
@@ -25,24 +25,26 @@ Currently, the {product} Operator does not support creating a `ServiceMonitor` c
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  name: <custom_resource_name> # <1>
-  namespace: {my-product-namespace} # <2>
+  name: <service-monitor-name> <1>
+  namespace: {my-product-namespace} <2>
   labels:
-    app.kubernetes.io/instance: <custom_resource_name>
-    app.kubernetes.io/name: backstage
+    app.kubernetes.io/instance: {my-product-cr-name} <3>
+    app.kubernetes.io/name: {product-custom-resource-type}
 spec:
   namespaceSelector:
     matchNames:
-      - {my-product-namespace}
+      - {my-product-namespace} <2>
   selector:
     matchLabels:
-      rhdh.redhat.com/app: backstage-<custom_resource_name>
+      app.kubernetes.io/instance: {my-product-cr-name} <3>
```

Comment:
This value differs from the name of the `ServiceMonitor` resource (`metadata.labels.app.kubernetes.io/instance`) we are creating. It should be the name of the RHDH deployment. We're trying to select the RHDH deployment to target the monitoring to.